lib/documents/schemas/sfo_cases.json

```diff
@@ -34,6 +34,14 @@
           "value": "closed"
         }
       ]
+    },
+    {
+      "key": "sfo_case_opened_date",
```

Comment:
Similarly, include these checks in the question, around the display metadata, filterable, preposition etc.

Reply:
Have checked with the user on the filterable and metadata issue. The preposition defaults to 'Related to'. In this case "Related to after(date)" and "Related to before (date)". 

I will change it to make it clearer it relates to cases opened